aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -492,6 +492,32 @@ The ASP.NET Core Module installer runs with the privileges of the **SYSTEM** acc
 1. Export the updated *applicationHost.config* file to the share.
 1. Re-enable the IIS Shared Configuration.
 
+::: moniker range=">= aspnetcore-2.2"
+
+## Application Initialization
+
+[IIS Application Initialization](/iis/get-started/whats-new-in-iis-8/iis-80-application-initialization) is an IIS feature that sends a fake HTTP request to the app. The request triggers the app to start. Application Initialization can be used by both the [in-process hosting model](xref:fundamentals/servers/index#in-process-hosting-model) and [out-of-process hosting model](xref:fundamentals/servers/index#out-of-process-hosting-model) with the ASP.NET Core Module version 2.
```

Comment:
is an IIS feature that sends a ~fake~ HTTP request to the app. 

No such thing as a fake request. Won't MT.

Reply:
Yeah ... I was a bit worried about that, too. I'll touch it up.

Reply:
"fake" is a word that MS engineers apply to **_my programming skills_**. 😀 lol